src/main/java/com/rackspacecloud/metrics/ingestionservice/influxdb/providers/CreateInfluxDB.java

```diff
@@ -0,0 +1,53 @@
+package com.rackspacecloud.metrics.ingestionservice.influxdb.providers;
+
+import org.influxdb.InfluxDBFactory;
+import com.rackspacecloud.metrics.ingestionservice.influxdb.config.BackupProperties;
+import org.influxdb.InfluxDB;
+import org.influxdb.dto.Query;
+import org.influxdb.dto.Pong;
+import org.influxdb.dto.QueryResult;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import com.rackspacecloud.metrics.ingestionservice.influxdb.providers.RestoreDBInfluxDB;
+import java.io.IOException;
+import com.rackspacecloud.metrics.ingestionservice.influxdb.providers.NotcreatedDBInfluxDB;
+//import org.springframework.kafka.test.context.EmbeddedKafka;
+//@ActiveProfiles(value = { "test" })
+//@EmbeddedKafka(partitions = 1, topics = { IngestionServiceApplicationTests.UNIFIED_METRICS_TOPIC })
+@EnableConfigurationProperties(BackupProperties.class)
+public class CreateInfluxDB {
+    // User name
+    private String username;
+    // Password
+    private String password;
+    // Connection address
+    private String openurl;
```

Comment:
What does the "open" part of `openurl` refer to? Perhaps call it "influxDbUrl" or "instanceUrl"